applications/FluidDynamicsApplication/custom_conditions/navier_stokes_wall_condition.cpp

```diff
@@ -363,6 +369,11 @@ const ConditionDataStruct& data)
         Vector rhs_gauss_behr = ZeroVector(TNumNodes*LocalSize);
         ComputeGaussPointBehrSlipRHSContribution( rhs_gauss_behr, data );
         rhs_gauss += rhs_gauss_behr;
+
+        // Maybe better with "if (FLAG == set){ etc. } ..."
+        Vector rhs_gauss_navier = ZeroVector(TNumNodes*LocalSize);
```

Comment:
What is pointed above also applies for the RHS.

Reply:
Done by using "+=".